### ISSUE TITLE
[ci] Migrate ROM E2e nightly run to CI

### ISSUE THREAD

**Issue by alphan** [lowRISC/opentitan#16715]

**Comment by alphan** [comment]
Moving nits from #16813 here:
- [ ] https://github.com/lowRISC/opentitan/pull/16813#discussion_r1050053510
- [ ] https://github.com/lowRISC/opentitan/pull/16813#discussion_r1050054060

**Comment by drewmacrae** [comment]
BTW it looks like if your bazel server hasn't shutdown this might splice from the bitstream for a previous commit

**Comment by milesdai** [comment]
> BTW it looks like if your bazel server hasn't shutdown this might splice from the bitstream for a previous commit

Are you referring to #16813? That commit should be introducing a pipeline that starts from a new container each time. Could you clarify the problem?

**Comment by drewmacrae** [comment]
Yeah, I think you're right @milesdai, docker is going to keep bazel from using a bazel server that's invalidated by the checkout.

**Comment by alphan** [comment]
> BTW it looks like if your bazel server hasn't shutdown this might splice from the bitstream for a previous commit

I wasn't bitten by this, maybe because I'm running this every 24h (> 18h refresh threshold)?

**Comment by milesdai** [comment]
Adding on to Alphan's checklist (since I can't edit that comment):

- [ ] https://github.com/lowRISC/opentitan/pull/16813#discussion_r1050053510 (randomness of pipeline time)
- [ ] https://github.com/lowRISC/opentitan/pull/16813#discussion_r1050054060 (refactoring pipeline build variables into a template)
- [ ] Refactor the initial checkout task into a template
